runtime/browser/demo/recipes.js

```diff
@@ -8,8 +8,50 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-let recipes = [{
-  name: "Create shortlist with <product, ...> and suggest similar products from <person>'s wish list",
+let recipes = [
+{
+  name: "Create shortlist with [product, ...]",
+  particles: [{
+    name: "Create",
+    constrain: {
+      "newList": "list"
+    }
+  },{
+    name: "SaveList",
+    constrain: {
+      "list": "list"
+    }
+  },{
+    name: "ShowProducts",
+    constrain: {
+      "list": "list"
+    }
+  }]
+}, 
+
+{
+  name: "See [person]'s wishlist",
```

Comment:
For the time being, the name fields help me see which recipe is which when editing/debugging. For that reason I'm hoping we can just leave them there for now.